test/e2e/controller/main_test.go

```diff
@@ -40,20 +40,20 @@ func TestMain(m *testing.M) {
 	)
 
 	if framework, err = e2eframework.NewFromFlags(); err != nil {
-		log.Printf("failed to setup framework: %v\n", err)
+		log.Errorf("failed to setup framework: %v", err)
```

Comment:
if we are using logrus, then we should do:

```suggestion
		log.WithError(err).Error("failed to setup framework")
```

No reason to use Errorf like before. Let's go all in. Same with below.